src/test/java/myCollections/MyArrayListTest.java

```diff
@@ -0,0 +1,58 @@
+package myCollections;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MyArrayListTest {
+
+    @Test
+    public void add() {
+        MyArrayList list = new MyArrayList<String>();
+        list.add("ss");
+        String actual = (String)list.get(0);
+        String expected = "ss";
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void remove() {
+        MyArrayList list = new MyArrayList<String>();
+        int espected = list.size();
+        list.add("ss");
+        list.remove(0);
+        int actual = list.size();
+        assertEquals(espected, actual);
+    }
+
+    @Test(expected = IndexOutOfBoundsException.class)
+    public void clear() {
```

Comment:
What do you test in this method?

Reply:
![image](https://user-images.githubusercontent.com/20267148/54884424-950a4b80-4e79-11e9-8dc7-7e6fffd22279.png)
It means you should check positive and negative scenarios.  And also use names that show what you test.